vnext/ReactUWP/ReactUWP.vcxproj

```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project DefaultTargets="Build" ToolsVersion="16.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$(SolutionDir)packages\Microsoft.Windows.CppWinRT.2.0.200316.3\build\native\Microsoft.Windows.CppWinRT.props" Condition="Exists('$(SolutionDir)packages\Microsoft.Windows.CppWinRT.2.0.200316.3\build\native\Microsoft.Windows.CppWinRT.props')" />
+  <Import Project="$(SolutionDir)packages\Microsoft.WinUI.3.0.0-alpha.200210.0\build\native\Microsoft.WinUI.props" Condition="Exists('$(SolutionDir)packages\Microsoft.WinUI.3.0.0-alpha.200210.0\build\native\Microsoft.WinUI.props')" />
```

Comment:
>Import [](start = 3, length = 6)

We should stop investing into the ReactUWP project development since we are deprecating it pretty soon.

Reply:
@vmoroz  I'm not convinced the change to this file is needed - I think I got confused because Microsoft.ReactNative is building cpp files in the ReactUWP directory. We should clean things up - are you already planning on doing that /  is there an issue filed for it?

Reply:
@vmoroz  We do have unit tests that rely on reactuwp.vcxproj:

React.Windows.Universal.UnitTests.vcxproj for example.

I'll keep this as is for now since I'm not planning on removing reactuwp.vcxproj in this PR. A follow up to remove it requires updating the tests and is orthogonal to WinUI 3 work.